Improve SAML mapper configuration descriptions

jvlcek opened the pull request:
https://bugzilla.redhat.com/show_bug.cgi?id=1438545

This PR expands upon the descriptions regarding mapper configurations
for SAML. It also removes a section that is no longer needed.

jvlcek commented:
@abellotti and @gtanzillo please review

jvlcek commented:
@miq-bot add_label bug

abellotti commented:
LGTM!!  Thanks @jvlcek for enhancing our SAML docs.

jvlcek commented:
Thank you @abellotti 

 @adahms when you get a chance, can you review/merge. Thanks!!

adahms commented on auth/saml.adoc:
```diff
@@ -182,12 +180,12 @@ to be SAML aware and function accordingly.
 return the user to the Appliance login screen allowing them to login as admin unless _Disable Local Login_
 is checked below.
+* Check: _Enable SAML_ - This will enable the SAML login button on the login screen, the redirects
+to the SAML protected page for authentication as well as supporting the SAML logout process.
```
Recommend changing 'authentication as well as supporting' to 'authentication, and supports' to improve clarity.

jvlcek replied:
Thank you @adahms again. Another good suggestions. Will do

adahms commented on auth/saml.adoc:
```diff
@@ -182,12 +180,12 @@ to be SAML aware and function accordingly.
 redirect to the SAML Identity Provider authentication screen.  Note that logouts from the Appliance will
 return the user to the Appliance login screen allowing them to login as admin unless _Disable Local Login_
 is checked below.
+* Check: _Enable SAML_ - This will enable the SAML login button on the login screen, the redirects
```
s/This will enable/This enables/

jvlcek replied:
Thank you @adahms. Good suggestions. Will do

jvlcek commented:
@adahms I've addressed your comments and squashed my commits down to one. Please let me know if you spot other issues you would like me to address. Thank you for the help!

JoeV

jvlcek commented:
@adahms Bump. Can this merged?

adahms commented:
@jvlcek - Looks good to me; merging!
